src/mscorlib.WinRT-Facade/ref/TypeForwards.cs

```diff
@@ -19,7 +19,7 @@
 [assembly: TypeForwardedTo(typeof(System.IntPtr))]
 [assembly: TypeForwardedTo(typeof(System.MulticastDelegate))]
 [assembly: TypeForwardedTo(typeof(System.Object))]
-#if !NETSTANDARD10 && !NETSTANDARD12
+#if !NETSTANDARD1_0 && !NETSTANDARD12
```

Comment:
What about NETSTANDARD1_1 up to 1_6?

Reply:
Nice catch! I'm going to scan the codes again and replace those.

Reply:
As far as I know, I have replaced the NETSTANDARD11, NETSTANDARD12 and NETCOREAPP30 and removed the DefineConstants. 
Please review 🙂

Reply:
Also looks like there are no occurrences of netstandard other than already identified netstandard11, 12, 21. 
I have searched the code base.